src/sentry/static/sentry/app/components/charts/releaseSeries.jsx

```diff
@@ -151,7 +152,12 @@ class ReleaseSeries extends React.Component {
   }
 
   getReleaseSeries = releases => {
-    const {organization, router, tooltip} = this.props;
+    const {organization, router, tooltip, queryExtra} = this.props;
+
+    const query = {...queryExtra};
+    if (new Set(organization.features).has('global-views')) {
```

Comment:
```suggestion
    if (organization.features.includes('global-views')) {
```

You can skip the Set() creation.